docker/Dockerfile.nvidia

```diff
@@ -90,7 +90,7 @@ RUN echo "$HPCSDK_HOME/cuda/lib" >> /etc/ld.so.conf.d/nvidia.conf && \
     echo "$HPCSDK_HOME/math_libs/lib64" >> /etc/ld.so.conf.d/nvidia.conf    
     
 # Compiler, CUDA, and Library paths
-ENV CUDA_HOME $HPCSDK_HOME/cuda
+ENV NVHPC_CUDA_HOME $HPCSDK_HOME/cuda
```

Comment:
I would keep both, CUDA_HOME is still used my most stuff for cuda path

Reply:
but then you would still get the warning?